source/Octo.Tests/Commands/CommandDesignFixture.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using FluentAssertions;
+using NUnit.Framework;
+using Octopus.Cli.Commands;
+using Octopus.Cli.Infrastructure;
+
+namespace Octo.Tests.Commands
+{
+    [TestFixture]
+    public class CommandDesignFixture
```

Comment:
We usually use `Convention` for these kind of tests
```suggestion
    public class CommandConventionFixture
```